cmd/easeprobe/main.go

```diff
@@ -31,17 +32,24 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+func getEnvOrDefault(key, defaultValue string) string {
+	if value, ok := os.LookupEnv(key); ok {
+		return value
+	}
+	return defaultValue
+}
+
+func isExternalLink(value string) bool {
+	_, err := url.Parse(value)
+	return err == nil
+}
+
```

Comment:
do you still need this function in `main.go`? as I've seen a similar function is `conf.,go`